### PR TITLE
MGMT-14078: Add "unavailable" option to the support-level enum

### PR DESCRIPTION
**Pull request by eliorerz**
## List all the issues related to this PR

- [x] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [ ] No tests needed

/cc @eliorerz

### CODE REVIEW

File: internal/featuresupport/features.go [hunk 1]
```diff
@@ -172,10 +172,6 @@ func (feature *ClusterManagedNetworkingFeature) GetName() string {
 }
 
 func (feature *ClusterManagedNetworkingFeature) GetSupportLevel(filters SupportLevelFilters) models.SupportLevel {
-	if swag.StringValue(filters.CPUArchitecture) == models.ClusterCPUArchitectureS390x || swag.StringValue(filters.CPUArchitecture) == models.ClusterCPUArchitecturePpc64le {
```
Comment (gamli75) [inline internal/featuresupport/features.go:175]: why did you remove this code? it should be unavailable now

Comment (eliorerz) [reply internal/featuresupport/features.go:176]: fixed
